conda.recipe/lux-widget/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "lux-widget" %}
+{% set version = "0.1.3" %}
```

Comment:
Similiarly, this way the version can be read in from [_version.py](https://github.com/lux-org/lux-widget/blob/master/luxwidget/_version.py) or package.json